project/accounts/templates/accounts/register/register.html

```diff
@@ -1,7 +1,7 @@
 {% extends "accounts/register/entry_base.html" %}
 
-{% block title %}CiviWiki - Register{% endblock %}
+{% block page_title %}Register{% endblock %}
 
-{% block content %}
+{% block entry_content %}
```

Comment:
This should remain as `content`
```suggestion
{% block content %}
```